ui/app/components/lifecycle-chart-row.js

```diff
@@ -26,6 +26,10 @@ export default class LifecycleChartRow extends Component {
 
   @computed('task.lifecycleName')
   get lifecycleLabel() {
+    if (!this.task) {
+      return '';
+    }
+
```

Comment:
This seems unrelated. How'd you end up here?

Reply:
I don’t totally understand but I was experiencing test failures when navigating to some search results and had to add a couple of null-handlings, I suspect it has something to do with the way the Mirage models are being constructed. With the time constraint I sadly just added the guards without digging further to grasp the underlying issue 😞

Reply:
That's reasonable!